charts/tidb-drainer/values.yaml

```diff
@@ -5,6 +5,10 @@
 # timezone is the default system timzone
 timezone: UTC
 
+# Change the name of the statefulset and pod
+# The default is clusterName-ReleaseName-drainer
```

Comment:
Can we warn user that changing name on existing will cause data loss?

Reply:
cc @gregwebs add a notice then we merge this?

Reply:
okay, warning added